### PR TITLE
Add a test script

### PR DESCRIPTION
**Pull request by djmetzle**
To confirm our implementation is correct, and debug improvements.

### CODE REVIEW

**Comment by djmetzle** [comment]
dev_block 👎 on finishing this

**Comment by g1y** [comment]
So I was perusing the rubygems docs, and found this: http://guides.rubygems.org/patterns/

Executables commonly go in bin/ it would seem. might be good to put this there.

File: test.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require "./lib/api.rb"
+
+require 'pp'
+
+api_token = ENV['REPLICATED_API_TOKEN']
+app_id = ENV['REPLICATED_APP_ID']
```
Comment (g1y) [inline test.rb:6]: Doesn't look like you're using this, it's being overwritten in the map below.

Comment (djmetzle) [reply test.rb:6]: True

**Comment by djmetzle** [comment]
Closing this. We should probably just do real unit testing. Branch can hang around for posterity though.

**Pull request closed without merging**